codes/block_definitions/utilities/argument_types.py

```diff
@@ -642,9 +642,12 @@ def __init__(self, value=None):
 
 
     def mutate(self):
-        choices = [100, 200, 300, 400, 500]
-        if self.value in choices:
-            choices.remove(self.value) # works in-place
+        # choices = [100, 200, 300, 400, 500]
```

Comment:
Temporary for testing (same with the changes below). Will change these back when running ezCGP for real.